packages/jasmine/src/package.json

```diff
@@ -17,11 +17,13 @@
     ],
     "main": "index.js",
     "dependencies": {
-        "jasmine": "~3.5.0",
-        "jasmine-core": "~3.5.0",
         "jasmine-reporters": "~2.3.2",
         "v8-coverage": "1.0.9"
     },
+    "peerDependencies": {
+        "jasmine": "~3.5.0",
+        "jasmine-core": "~3.5.0"
```

Comment:
I don't think we need to list this one since `jasmine` depends on `jasmine-core`

Reply:
unfortunately since in jasmine_runner we require `jasmine-core` it must be specified as a dep, even though it's a transitive dep of `jasmine`, if it's not specified has a `dependencies` and `peerDependencies` then yarn wont hoist it to the top.
You can see the result of this in the CI failures
```
==================== Test output for //:test:
Error: Cannot find module 'jasmine-core'. Please verify that the package.json has a valid "main" entry
    at Function.module.constructor._resolveFilename (test_require_patch.js:482:17)
    at Function.Module._load (internal/modules/cjs/loader.js:687:27)
    at Module.require (internal/modules/cjs/loader.js:849:19)
```

Reply:
So either we figure out how to remove the require tto `jasmine-core` or we have to put `jasmine-core` in our `peerDependencies`

Reply:
got it, thanks for the explanation. we should add that as a comment here.